lib/src/migrators/module/scope.dart

```diff
@@ -14,6 +14,8 @@ import 'package:sass/src/ast/sass.dart';
 import 'package:path/path.dart' as p;
 import 'package:source_span/source_span.dart';
 
+enum UnreferencableType { localFromImporter, globalFromNestedImport }
```

Comment:
It's generally good style to use pseudo-enums (such as Dart Sass's [`Syntax`](https://github.com/sass/dart-sass/blob/master/lib/src/syntax.dart)) rather than Dart enums, because they're easier to give extra behavior to and control the stringification of.